x-pack/plugin/ml-package-loader/src/main/java/org/elasticsearch/xpack/ml/packageloader/action/TransportLoadTrainedModelPackage.java

```diff
@@ -190,7 +190,7 @@ public TaskId getParentTask() {
 
             @Override
             public CancellableTask createTask(long id, String type, String action, TaskId parentTaskId, Map<String, String> headers) {
-                return new CancellableTask(id, type, action, getDescription(), parentTaskId, headers);
+                return new CancellableTask(id, type, action, "modelId=" + request.getModelId(), parentTaskId, headers);
```

Comment:
This is needed so we can search for a particular task. We could also put this in the `action` field but when I looked around it didn't seem like that was a common place to do this.

Reply:
TODO: move this to a common place so that the request knows how to build it